SplunkRumWorkspace/SplunkRum/SplunkRum/Session.swift

```diff
@@ -62,6 +68,16 @@ func getRumSessionId() -> String {
     for callback in callbacks {
         callback()
     }
-
+    if isSessionIdChanged {
+        createSessionIdChangeSpan()
```

Comment:
Could we invoke this right when expiration is checked? Can get rid of `isSessionIdChanged` then

Reply:
As per our discussion, we keep it as it is.